src/Hedgehog/Gen.fs

```diff
@@ -409,7 +409,7 @@ module Gen =
 
     /// Generates a random 64-bit floating point number.
     let double : Gen<double> =
-        create Shrink.double Random.sizedDouble
+        create (Shrink.towardsDouble 0.0) Random.sizedDouble
```

Comment:
I know that at some point we need to change `Gen.double` so that it takes a `Range`, similar to the other primitive combinators. I'd prefer to do this on a separate PR because `Random.sizedDouble` must be changed too and `Seed.nextDouble` must be added first. (Same also for `Gen.float`.)